src/index.js

```diff
@@ -44,10 +44,17 @@ const WebRTCStar = or(
   and(WebSocketsSecure, base('p2p-webrtc-star'), base('ipfs'))
 )
 
-const WebSocketsStar = or(
-  and(WebSockets, base('p2p-websockets-star')),
-  and(WebSockets, base('p2p-websockets-star'), base('ipfs')),
-  and(WebSocketsSecure, base('p2p-websockets-star'), base('ipfs'))
+const WebSocketStar = or(
+  or(
+    or(
+      and(WebSockets, base('p2p-websocket-star'), base('ipfs')),
+      and(WebSocketsSecure, base('p2p-websocket-star'), base('ipfs'))
+    ),
+    or(
+      and(WebSockets, base('p2p-websocket-star')),
+      and(WebSocketsSecure, base('p2p-websocket-star'))
+    )
+  )
```

Comment:
These 4 `or` can be just one `or`

Reply:
There is a bug. When `and(WebSocketsSecure, base('p2p-websocket-star'), base('ipfs'))` comes after `and(WebSockets, base('p2p-websocket-star'))` the tests fail. At the beginning it was like that and I thought that adding 4 ors would fix the bug. Now realized reordering does fix it too.